docs/data/material/migration/upgrade-to-v6/upgrade-to-v6.md

```diff
@@ -30,6 +30,7 @@ If you're using any of these packages, you can also change their version to `"6.
 - `@mui/lab`
 - `@mui/material-nextjs`
 - `@mui/styled-engine-sc`
+- `@mui/styles`
```

Comment:
Yeah, we probably can have this, but do we expect people using v5 to still depend on this package? Maybe we need a dedicated v4 -> v6 migration strategy. cc @DiegoAndai

Reply:
The issue asking for this has only had two upvotes so far: https://github.com/mui/material-ui/issues/42676, so I wouldn't use resources for it at this moment.

Maybe we can add a callout here to point v4 users to the v4 -> v5 migration, and also link the issue (https://github.com/mui/material-ui/issues/42676) so they can upvote it?

Reply:
Sounds good to me Diego